resolver/resolver.go

```diff
@@ -168,6 +168,9 @@ type BuildOptions struct {
 	// field. In most cases though, it is not appropriate, and this field may
 	// be ignored.
 	Dialer func(context.Context, string) (net.Conn, error)
+	// Authority is the effective authority of the connection for which the
+	// resolver is built for.
```

Comment:
`client` or `ClientConn` instead of `connection`.

And: `for which the resolver is built.`

Reply:
Done.